test/e2e/dcos/dcos.go

```diff
@@ -235,6 +235,9 @@ func (c *Cluster) AppCount() (int, error) {
 // Version will return the node count for a dcos cluster
 func (c *Cluster) Version() (string, error) {
 	out, err := c.Connection.Execute("curl -s http://localhost:80/dcos-metadata/dcos-version.json")
+	if err != nil {
```

Comment:
Is this code related?

Reply:
we were getting lint warning ...

Reply:
Nah, it's fine, strange that it should choose this PR to complain about that!

Reply:
So my CI build failed with make test-style. And I fixed all the things it complained about. This showed up as a warning. And my guess is that it has been ignored all this while. Also that check is needed. We are calculating err and not using it.